### PR TITLE
polish(fir): polish ensure_softmax function

### PR DESCRIPTION
**Pull request by puyuan1996**
- The check to determine whether the input logits are already normalized in the function ensure_softmax is computationally expensive and appears to be rarely necessary.
- For further details, please refer to https://github.com/opendilab/LightZero/issues/383.

### CODE REVIEW

**Review by Firerozes — COMMENTED** [review]
Great !
